src/components/Breadcrumb/LinkRouter.tsx

```diff
@@ -0,0 +1,13 @@
+import Link, { LinkProps } from "@mui/material/Link";
+import {
+  Link as RouterLink
+} from "react-router-dom";
+
+interface LinkRouterProps extends LinkProps {
```

Comment:
Using type picking seems better here because the `to` property in the react-router `Link` component is not a string
```suggestion
type LinkRouterProps = LinkProps & Pick<ReactRouterLinkProps, "to" | "replace">
```